ruby-sdk/lib/eppo_client/config.rb

```diff
@@ -6,14 +6,15 @@
 module EppoClient
   # The class for configuring the Eppo client singleton
   class Config
-    attr_reader :api_key, :assignment_logger, :base_url, :poll_interval_seconds, :poll_jitter_seconds
+    attr_reader :api_key, :assignment_logger, :base_url, :poll_interval_seconds, :poll_jitter_seconds, :log_level
 
-    def initialize(api_key, assignment_logger: AssignmentLogger.new, base_url: EppoClient::Core::DEFAULT_BASE_URL, poll_interval_seconds: EppoClient::Core::DEFAULT_POLL_INTERVAL_SECONDS, poll_jitter_seconds: EppoClient::Core::DEFAULT_POLL_JITTER_SECONDS, initial_configuration: nil)
+    def initialize(api_key, assignment_logger: AssignmentLogger.new, base_url: EppoClient::Core::DEFAULT_BASE_URL, poll_interval_seconds: EppoClient::Core::DEFAULT_POLL_INTERVAL_SECONDS, poll_jitter_seconds: EppoClient::Core::DEFAULT_POLL_JITTER_SECONDS, initial_configuration: nil, log_level: "info")
       @api_key = api_key
       @assignment_logger = assignment_logger
       @base_url = base_url
       @poll_interval_seconds = poll_interval_seconds
       @poll_jitter_seconds = poll_jitter_seconds
+      @log_level = log_level
```

Comment:
I think it would be helpful to have a few unit tests for the log levels

Reply:
It's a bit hard to test in general because apps usually don't have access to their stderr